### PR TITLE
add watch task for easier hacking

### PR DESCRIPTION
**Pull request by foxyblocks**

### CODE REVIEW

**Comment by spikebrehm** [comment]
Sweet, thanks @wordofchristian! I've been meaning to add this. I can't seem to get it to work locally though :(. Do you see it working?

**Comment by spikebrehm** [comment]
Oh durr, I forgot to run `grunt watch`.  It would be awesome to have it also restart the Node server when this happens so you can refresh without restarting.  Is there a typical `grunt watch` workflow you use?

**Comment by foxyblocks** [comment]
Yeah, I was using it for client code which works now. Basically it keeps you from having to run the compile task every time you make a change. I hadn't tested it with server changes, but it should be doable with grunt-shell or something like that. Updating now...

**Comment by foxyblocks** [comment]
That seems to do it. I'm not sure about keeping the grunt commands in the index.js file though. Seems roundabout.

File: Gruntfile.js [hunk 1]
```diff
@@ -41,6 +48,23 @@ module.exports = function(grunt) {
...
+      scripts: {
+        files: 'app/**/!(compiledTemplates).js',
+        tasks: ['rendr_stitch'],
+        options: {
+          interrupt: true
+        }
+      },
+      templates: {
+        files: 'app/**/*.hbs',
+        tasks: ['handlebars', 'rendr_stitch'],
```
Comment (spikebrehm) [inline Gruntfile.js:61]: Is it better to just have this run `handlebars`, which updates `compiledTemplates.js`, which runs `rendr_stitch`?  Seems cleaner.

Comment (foxyblocks) [inline Gruntfile.js:61]: True, but at the time it seemed too implicit. Up to you...

**Comment by spikebrehm** [comment]
Dude, this is awesome! I've been meaning to get around to this for so long but this does just what I needed.

File: Gruntfile.js (hunk 1 above)
Comment (spikebrehm) [inline Gruntfile.js:61]: Word, I made that change: https://github.com/airbnb/rendr-app-template/commit/1b5f16e21e61158e4b4cc06d0a87fc4833092983

**Comment by spikebrehm** [comment]
Just moved the call to `grunt compile` to `grunt server` from `index.js`, so there's less delay between `nodemon` restarts. This is hot. I'll have to add it to our other Rendr projects.

**Comment by spikebrehm** [comment]
And stylesheets! https://github.com/airbnb/rendr-app-template/commit/3b7095536de7f451271d0e0a95d53c164c228c50

**Pull request merged**